js/index.js

```diff
@@ -0,0 +1,33 @@
+/*!
+ * Copyright (c) 2023 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const context = require('./context');
+
+const constants = {
+  // default context output filename under ./contexts/
+  CONTEXT_FILENAME: 'vdl-aamva-v1.jsonld',
+  // default context well known URL id
+  CONTEXT_URL: 'https://w3id.org/vdl/aamva/v1',
+  // default context CBOR-LD codec id
+  // value between 0x0 and 0x7FFF for globally registered term codec values
+  // or >= 0x8000 for app-specific local terms
+  CBORLD_VALUE: 0x0
```

Comment:
We should probably pick a proper value here or not expose this -- otherwise someone will use it. (I'm presuming `0x0` isn't a proper value here).

Reply:
Zero is reserved.  We should probably make zero in particular mean to not use those magic values.

Reply:
@davidlehn I noticed that this discussion was revisited in the OpenCred Slack channel. Do we have an idea about what this value will be for VDL AAMVA?